Makefile

```diff
@@ -325,6 +325,8 @@ GOCFLAGS :=
 GOCFLAGS-y :=
 RUSTCFLAGS :=
 RUSTCFLAGS-y :=
+CARGOFLAGS :=
+CARGOFLAGS-y :=
```

Comment:
Do we really need cargo in the core repository? I'd prefer to have it part of an external language support library.